### PR TITLE
[9.0] [MIG] website_sale_require_legal module 

### PR DESCRIPTION
**Pull request by cubells**
* Updated README.rst

@Tecnativa

### CODE REVIEW

File: website_sale_require_legal/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Tecnativa - Jairo Llopis
+# Copyright 2016 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Require accepting legal terms to checkout",
+    "summary": "Force the user to accept legal tems to buy in the web shop",
+    "version": "9.0.1.1.0",
```
Comment (pedrobaeza) [inline website_sale_require_legal/__openerp__.py]: 9.0.1.0.0

File: website_sale_require_legal/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================================
+Require accepting legal terms to checkout
+=========================================
+
+This module was written to extend the functionality of your website shop to
+support forcing the user to accept your legal advice, terms of use and privacy
+policy, and allow you to obey some countries' laws.
```
Comment (pedrobaeza) [inline website_sale_require_legal/README.rst]: s/obey/comply with

File: website_sale_require_legal/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "category": "Website",
+    "website": "http://www.tecnativa.com",
+    "author": "Antiun Ingeniería S.L., "
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
```
Comment (pedrobaeza) [inline website_sale_require_legal/__openerp__.py]: Remove empty keys.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Please rebase on current 9.0 branch, because I have fixed .travis.yml for allowing sass command.

**Comment by cubells** [comment]
Is Travis error related to another module?

**Comment by pedrobaeza** [comment]
Travis is failing due to website_sale_wishlist module, so not related.

**Comment by pedrobaeza** [comment]
Please rebase again, as sass command is still not working.

**Comment by yajo** [comment]
This addon [breaks a tour in `website_sale_wishlist`](https://travis-ci.org/OCA/e-commerce/jobs/180007434#L1000-L1008). You have to fix that tour. It might not be an easy task for tour beginners, should I do that @pedrobaeza?

**Comment by pedrobaeza** [comment]
The error is also on base branch, so it's not something of this branch. @Yajo, please fix it on another PR.

**Comment by yajo** [comment]
I hope #149 and https://github.com/OCA/server-tools/pull/666 fix it.

**Comment by yajo** [comment]
You need to rebase.

**Review by yajo — APPROVED** [review]
There's a coverage drop, but code & functional test passed.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**